cypress/fixtures/mock-country.json

```diff
@@ -0,0 +1,106 @@
+{
+  "country": {
+      "data": {
+          "id": "3",
+          "type": "country",
+          "attributes": {
+              "name": "Brazil",
+              "description": "Pão de queijo is a famous Brazilian cheese bread made with cassava flour and cheese, resulting in a chewy and cheesy delight."
+          },
+          "relationships": {
+              "breads": {
+                  "data": [
+                      {
+                          "id": "13",
+                          "type": "bread"
+                      },
+                      {
+                          "id": "14",
+                          "type": "bread"
+                      },
+                      {
+                          "id": "15",
+                          "type": "bread"
+                      },
+                      {
+                          "id": "16",
+                          "type": "bread"
```

Comment:
We have hand-rolled and removed this from our Serialized JSON payload. Just getting rid of information that we don't need to send and stuff you don't need to see. #efficiency